packages/backend/src/app.ts

```diff
@@ -144,6 +145,19 @@ export type HttpSigContext = AppContext & {
   client: string
 }
 
+type TenantedHttpSigHeaders = HttpSigHeaders & Record<'tenantId', string>
+
+type TenantedHttpSigRequest = Omit<HttpSigContext['request'], 'headers'> & {
+  headers: TenantedHttpSigHeaders
+}
+
+export type TenantedHttpSigContext = HttpSigContext & {
+  headers: TenantedHttpSigHeaders
+  request: TenantedHttpSigRequest
+  tenant?: Tenant
+  isOperator: boolean
+}
```

Comment:
These types seem to be more suited for the Open Payments server. The main thing we need to do here is to add the the tenant object to the GraphQL context (`ApolloContext` & lines 409 -> 418), such that we can get the tenant information in the resolvers